### PR TITLE
Add Visa Argentina

### PR DESCRIPTION
**Pull request by fainpablo**

### CODE REVIEW

**Comment by phallobst** [comment]
Visa should rather go into "Payments".

**Comment by fainpablo** [comment]
I thought the same, but why AMEX is in the "Banking" category?

**Comment by fainpablo** [comment]
I thought the same, but AMEX in the "Banking" category got me confused.

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
AmEx was added back in 2014 (#38) when there was only one category. My guess is that it got moved to the bank category because there wasn't any payments category yet and no one have thought about it since that.

Please move Visa to the payments section.

**Comment by fainpablo** [comment]
Makes sense. I've just moved Visa to payments. Will submit a new PR to update AMEX.

File: _data/payments.yml [hunk 1]
```diff
@@ -210,6 +210,11 @@ websites:
       - sms
     doc: https://help.venmo.com/hc/en-us/articles/217532397
 
+  - name: Visa Argentina
+    url: https://www.visa.com.ar/
+    img: visa.png
+    twitter: VisaArgentina
```
Comment (phallobst) [inline _data/payments.yml:216]:
```suggestion
    twitter: VisaArgentina
    lang: es
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]
I approve your visa!

**Pull request merged**